tensorflow/lite/java/demo/lite_demo

```diff
@@ -0,0 +1 @@
+
```

Comment:
Why this one is required? And also please provide the url of the file changed.  
For CLA, please sign in the cla with all email ids you are signed in your computer.